app/models/bundle.rb

```diff
@@ -5,23 +5,13 @@ class Bundle < Product
 
   def products_for_group_select
     products = facility.products.where.not(type: "Bundle").order(:type, :name)
-    current_group = []
-    current_opts  = []
-    groups = []
+    groups = products.map {|x| x.class.name.pluralize}
```

Comment:
Is there a way we could use `group_by { |p| p.class.name.pluralize }`? It might make the conversion to `[p.to_s_with_status, p.id]` a little trickier, though.